### ISSUE TITLE
should I finetune monodepth2 model ?

### ISSUE THREAD

**Issue by a444447** [Gandolfczjh/3D2Fool#9]
Hello,
If I want to achieve the same depth map quality as shown in the paper, do I need to fine-tune the Monodepth2 model first, or can I simply use the pre-trained model provided by Monodepth2?

**Comment by Gandolfczjh** [comment]
Yes, you can use CARLA to create a simple monocular depth estimation training dataset and fine-tune these MDE models. However, even without fine-tuning, the effect of the texture attack during training would not differ significantly.
